### ISSUE TITLE
ITI-65: Make Folders "add only" like they are in XDS

### ISSUE THREAD

**Issue by jlamy** [IHE/ITI.MHD#52]
**Section Number** *11.4.1.2*

**Issue** *In XDS, Document Entries may only be added to Folders, not removed. Because the association is a separate object, each FD-DE association is immutable, even after the DE is deprecated. However, in FHIR the List resource allows items to be added or removed from it.*

**Proposed Change** *We should consider constraining Folder List.entry.item in MHD to be "add only" to match the XDS semantics, and allow the MHD Document Recipient to ignore removals without error. This would have the added benefit of reducing the impact of resource contention in competing updates to Lists without needing to support versioning and If-Match, which is problematic (see [here](https://jira.hl7.org/browse/FHIR-31925)).*

**Priority**: Medium

**Comment by JohnMoehrke** [comment]
I agree. Is there text in XDS where this folder behavior can be referenced?
